### PR TITLE
Completed html-forms exercise

### PR DESCRIPTION
**Pull request by 16hsuisaac**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<h1>Coding Form!</h1>
+<form action="" method="GET" class="form">
+  <div>
+    <label for="email">Email</label><br>
+      <input type="text" id="email" name="email">
+
+  </div>
+  <div>
+    <label for="coding">Why is coding cool?</label><br>
+    <select name="coding">
```
Comment (codingcodymiller) [inline html-forms/index.html:17]: This `select` will need an `id` to match the `for` attribute on its `label`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! One note for you:

**Comment by 16hsuisaac** [comment]
html-forms

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**